Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -296,11 +296,23 @@ extension PackageShow.Model {
     }
 
     func authorsListItem() -> Node<HTML.ListContext> {
-        guard let authors = authors else { return .empty }
-        let nodes = authors.map { Node<HTML.BodyContext>.a(.href($0.url), .text($0.label)) }
+        guard Environment.current == .development else { return .empty }
+        guard let authors else { return .empty }
+        var nodes = authors.authors.map { author -> Node<HTML.BodyContext> in
+            if let authorURL = author.url {
+                return .a(.href(authorURL), .text(author.name))
+            } else {
+                return .text(author.name)
+            }
+        }
+
+        if (authors.numberOfContributors > 0) {
```

Comment:
nit: no need for brackets around the if statement

```suggestion
        if authors.numberOfContributors > 0 {
```